tutorials/advanced/elasticSnorkel.ipynb

```diff
@@ -0,0 +1,583 @@
+    "\n",
+    "Elasticsearch uses slightly different terminology from tradational databases. \n",
+    "\n",
+    "This is the following mapping:\n",
+    "Database = Index\n",
+    "Table = Type\n",
+    "Row = Document\n",
+    "\n",
+    "Here we intialize an object where corpus will be the name of the index with a single type articles, and a field named sentence."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from snorkel import SnorkelSession\n",
+    "from snorkel.models import Document, Sentence\n",
+    "from elastics import elasticSession ,printResults,deleteIndex\n",
+    "\n",
+    "# import os\n",
+    "\n",
+    "# # TO USE A DATABASE OTHER THAN SQLITE, USE THIS LINE\n",
+    "# # Note that this is necessary for parallel execution amongst other things...\n",
+    "# os.environ['SNORKELDB'] = 'postgres:///snorkel-intro'\n",
+    "\n",
+    "session = SnorkelSession()\n",
+    "eSearch=elasticSession(\"corpus\",\"articles\",\"sentence\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Check Connection\n",
+    "\n",
+    "We begin indexing, we can check our connection with Elasticsearch as well as our existing indices. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Index Information: \n",
+      " \n",
+      "health status index uuid pri rep docs.count docs.deleted store.size pri.store.size\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "eSearch.getIndices()\n",
+    "\n",
+    "#To delete an index by name or _all to delete all indices\n",
+    "# deleteIndex(\"indexName\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Indexing the Corpus\n",
+    "\n",
+    "Now that we have an established connection we can begin indexing. By default each document will contain a field corresponding to the sentence ID number, the sentence and an empty vector of 'o's (this will be useful for generating candidate tags below). Once everything has been indexed we can see its status with the amount of documents it contains as well as the size."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Beginning indexing\n",
+      "2591 items indexed\n",
+      "Index Information: \n",
+      " \n",
+      "health status index  uuid                   pri rep docs.count docs.deleted store.size pri.store.size\n",
+      "yellow open   corpus HCOENVAeRoKTrHduXdVChQ   5   1      67820            0       24mb           24mb\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "eSearch.elasticIndex(Document)\n",
+    "eSearch.getIndices()"
```

Comment:
Can this just be run automatically when the `elasticSession` object is created?  That would be way cleaner